UPGRADE-3.x.md

```diff
@@ -4,3 +4,10 @@ UPGRADE 3.x
 ## Deprecated Admin class
 
 The `Admin` class is deprecated. Use `AbstractAdmin` instead.
+
+## Deprecated template fallback mechanism
+
+The method that gets templates in the Twig extension loaded a default template
+when the one specified could not be found. You can no longer rely on that and
+should always specify templates that exist. If anyone knows why this system
```

Comment:
> If anyone knows why this system ...

Not sure it should be here.

If you want to keep it, it should be on a new line for a nicer rendering.